packages/frontend/src/components/navigation/NavLinks.js

```diff
@@ -118,7 +119,7 @@ const NavLinks = () => (
             <UserIcon/>
             <Translate id='link.account'/>
         </NavLink>
-        <a href='https://nearhelp.zendesk.com/' target='_blank' rel='noopener noreferrer' onClick={() => Mixpanel.track('Click Help button on nav')}>
+        <a href={isWhitelabel() ? 'support.mynearwallet.com' : 'https://nearhelp.zendesk.com/'} target='_blank' rel='noopener noreferrer' onClick={() => Mixpanel.track('Click Help button on nav')}>
```

Comment:
nit: can we do `https://support.mynearwallet.com`

Reply:
Done✅